frontend/.husky/commit-msg

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+if [ -z "${CI}" ] || [ "${CI}" != true ]; then
+  if ! command -v git-secrets >/dev/null 2>&1; then
+      echo "git-secrets is not installed. Please visit https://github.com/awslabs/git-secrets#installing-git-secrets"
+      exit 1
+  fi
+
+  git-secrets --register-aws > /dev/null
```

Comment:
issue(non-blocking): there are two issues here:
1. I'm not really sure `git secrets --install` can be dropped, are you sure it's only creating git hooks files?
1. `git-secrets --register-aws > /dev/null` is not a part of the hook, is the configuration of git-secrets, so in my opinion should not be re-done for every hook, but should be done just at git secrets installation. So if you install husky at `prepare` I suggest to put `git-secrets --register-aws` (and maybe all the if) there

Reply:
1. Apparently yes: https://github.com/awslabs/git-secrets#options-for-install
2. Had the same doubt. This approach is proposed by this [article](https://www.codewithyou.com/blog/using-husky-and-gitsecrets-to-prevent-committing-passwords-and-other-sensitive-information-to-a-git-repository). I can try to put in prepare and test again.

Reply:
suggestion(non-blocking): I'd suggest to group the shared logic under a single file to source in the various scripts

Reply:
I do not know if creates problems, but mind this line https://github.com/awslabs/git-secrets#options-for-install:~:text=Git%20only%20allows%20a%20single%20script%20to%20be%20executed%20per%20hook